paddle/fluid/inference/api/paddle_pass_builder.cc

```diff
@@ -331,6 +331,7 @@ void CpuPassStrategy::EnableMKLDNN() {
              // "mkldnn_inplace_pass",  // This pass should be activated after
              // fuses. Disabled by default due to
              // little gain and lots of problems
+             "operator_scale_onednn_fuse_pass",  //
```

Comment:
move it before the comment, this commented out `mkldnn_inplace_pass` can cause trouble if someone will uncomment it.

Reply:
Do you mind if I remove `mkldnn_inplace_pass` as we're probably not going to maintain it?